### ISSUE TITLE
Wrong Detected Results

### ISSUE THREAD

**Issue by hoangminhtoan** [fizyr/keras-retinanet#266]
Hi,
I run retinanet after updating #244 bug fix. It took me almost 1.5 days to run the training with default config (weight='imagenet', epoch=50, step/epoch=10000), and even I changed the weight='resnet50' I still got the same wrong result. Could someone explain which step I missed? Is there anyone who get the same result as me? I install kera-retinanet in a virtual environment, so I remove remark '--user' while installing from source code. It seems to detect the correct bounding box but wrong label Many thanks!!
![screenshot from 2018-02-02 10-35-40](https://user-images.githubusercontent.com/1559257/35712378-0cf975ba-0805-11e8-966f-faba8f30d4c2.png)
![screenshot from 2018-02-02 10-36-36](https://user-images.githubusercontent.com/1559257/35712379-0d201404-0805-11e8-8432-7ef1ca0138d9.png)
![screenshot from 2018-02-02 10-38-46](https://user-images.githubusercontent.com/1559257/35712415-45abdc54-0805-11e8-9fc4-da70b09bf98a.png)

**Comment by hgaiser** [comment]
That is really odd. Are you getting the same result if you use the pretrained network? Are you using the example to do the visualization?

I assume you trained on COCO ?

**Comment by hoangminhtoan** [comment]
No I did not use COCO, I trained PASCAL VOC 2007 from the beginning. And of course your code downloaded ‘resnet_...weight.h5’ file automatically.

**Comment by hgaiser** [comment]
Ah but the labels to class names for VOC is different. The example by default loads in the COCO labels to names. The mapping can be derived from [here](https://github.com/fizyr/keras-retinanet/blob/master/keras_retinanet/preprocessing/pascal_voc.py#L30-L51).
